### PR TITLE
Add distance matrix sample

### PR DESCRIPTION
**Pull request by gaelsimon**

### CODE REVIEW

**Review by shwetadeshmukh — APPROVED** [review]
LGTM 👍 except when I select Departure time from the datepicker, it always throws an error to me. You might want to check once at your end before merging.

**Comment by gaelsimon** [comment]
> LGTM 👍 except when I select Departure time from the datepicker, it always throws an error to me. You might want to check once at your end before merging.

Yes, it's because we have an issue on the API itself. should be fixed pretty soon

**Pull request merged**
